reposilite-plugins/javadoc-plugin/src/main/kotlin/com/reposilite/plugin/javadoc/DocExtractor.kt

```diff
@@ -0,0 +1,51 @@
+package com.reposilite.plugin.javadoc
+
+import java.io.File
+import java.io.IOException
+import java.nio.file.Path
+import java.util.jar.JarFile
+
+/**
+ * @author TOTHTOMI
+ * @version 1.0.0
+ * @since Dec. 27, 2021
+ */
+internal class DocExtractor {
+
+    @Throws(Exception::class)
```

Comment:
Error handling through exceptions is pretty much antipattern, sth like `Result<Unit, ErrorResponse/Exception>` might be a better choice